src/sql/azdata.proposed.d.ts

```diff
@@ -2940,6 +2940,7 @@ declare module 'azdata' {
 		value?: string | CategoryValue;
 		values?: string[] | CategoryValue[];
 		editable?: boolean;
+		fireOnTextChange?: boolean;
```

Comment:
you need to implement this in `src/sql/workbench/api/node/extHostModelView.ts`. The editable property is implemented there, see the DropDownWrapper class.

Also can we update sqlops.proposed.d.ts with this too as (for now) we aren't ready to update the SQL 2019 extension codebase to use the new azdata code.

Thanks!

Reply:
done